### DISCUSSION TITLE
Help with Redux Dev Tools

### DISCUSSION THREAD

**Discussion by toddmillernyc** [mrpmorris/Fluxor#504] [Q&A]
In both the sample application FluxorBlazorWeb.ReduxDevToolsTutorial.Server and my own app (.NET 8) I cannot get redux devtools to work.  I am using Chrome Version 128.0.6613.85 (Official Build) (64-bit) and have reinstalled redux dev tools. 

In the sample app, I increment the counter and open the weather page and redux dev tools says "no store found".  In the .NET 8 app, the  devtools icon is not highlighted in the toolbar, and I also get the "no store found" message.

csproj:
```
    <PackageReference Include="Fluxor.Blazor.Web.ReduxDevTools" Version="6.0.0" />
```
    Program.cs:
```   
    builder.Services.AddFluxor(options =>
{
    options.ScanAssemblies(typeof(Program).Assembly);
#if DEBUG
    options.UseReduxDevTools(reduxDevTools =>
    {
        reduxDevTools.Name = "ReconApp";
    });
#endif
});
```

**Comment by mrpmorris** [comment]
The default dev browser often disables plugins, ensure RDT plugin is enabled and refresh the page.
